website/docs/components/date-picker/date-a11y.md

```diff
@@ -7,31 +7,86 @@ tabs: Design('date-picker'), A11y('date-a11y'), API('date-api'), Example('date-c
+| `Enter` , `Space`           | Open the date picker dropdown. Move focus to selected date, specifically the date displayed in the date input text field. If no date has been selected, places focus on the current date. |
+| `ESC`                       | Closes the dialog and returns focus to the trigger.                                                                                                                                       |
+| `Up Arrow`, `Down Arrow`    | Moves focus to the same day of the previous or next week, respectively.                                                                                                                   |
+| `Left Arrow`, `Right Arrow` | Moves focus to the previous or next day, respectively.                                                                                                                                    |
+
+### Roles and attributes
+
+The following list describes roles and attributes that component already has.
+
+#### Trigger
+
+Table: Trigger roles and attributes
+
+| Component / element                         | Role / Attribute                                                               | Usage                                                                                                          |
+| ------------------------------------------- | ------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------- |
+| `DatePicker.Trigger`                        | `aria-label="Date field"`                                                      | Defines the accessible name.                                                                                   |
+|                                             | `aria-controls="IDREF"`                                                        | Indicates which element this trigger opens.                                                                    |
+| <nobr>`DatePicker.Trigger` > `input`</nobr> | `aria-haspopup="dialog"` and inherits `aria-controls="IDREF"` from its parent. | `aria-haspopup` indicates that the element triggers a dialog.                                                  |
+|                                             | `aria-expanded="false/true"`                                                   | Indicates whether the dialog is open or not.                                                                   |
+|                                             | `role="combobox"`                                                              | Identifies the element as a combobox—a text input combined with a list of options.                             |
+|                                             | `aria-describedby="IDREF"`                                                     | Provides the input with an accessible description by linking to a hint that explains the required data format. |
```

Comment:
don't get this thing with the hint, in the basic example it refers to an empty span. Is it correct? How is it supposed to work?

Reply:
Yeah, for sure, it's strange. The date format "MM/DD/YYYY" should be moved from aria-label to that span

<img width="622" alt="image" src="https://github.com/user-attachments/assets/0be3c2d1-53a7-42f8-9062-a84b2b6ae91e">

Reply:
Guys, @ilyabrower @msereniti can we move date format to the span that should contain the date format?

Reply:
Michael fixed this ✅